src/harness.ts

```diff
@@ -47,6 +47,28 @@ const EVENT_HANDLERS = [
 ];
 
 /**
+ * An internal function which finds a VNode based on `key`
+ * @param target The VNode to search
+ * @param key The key to find
+ */
+function findVNodebyKey(target: VNode, key: string | object): VNode | undefined {
```

Comment:
I guess what would be the expected behaviour?  That the same event gets dispatched to all the identified nodes, or that the function throws, saying it could not resolve to a single node?  I had basically been considering _caveat emptor_ and noting in the README that it only finds the first `key` in what is a DFS algorithm, but I forgot to mention this in the README (it was in my head though, I promise).